lib/resorts/falls-creek/index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  selector: 'div[class=="snow-inner-table-content"]',
```

Comment:
'.snow-table-lift-status .snow-inner-table-content .table-row' selects every row in the table and from that you can try finding children with name and status
That said - this is again the case HTML does not have status information. Looks like you need to parse https://www.fallscreek.com.au/wp-content/uploads/FCSnowReport.json to get it. Use Mount Hotham as an example of how to do it - you'll have to change both 'index.js' and the test file.